pep-0693.rst

```diff
@@ -0,0 +1,101 @@
+PEP: 693
+Title: Python 3.12 Release Schedule
+Version: $Revision$
+Last-Modified: $Date$
+Author: Thomas Wouters <thomas@python.org>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 24-May-2022
+Python-Version: 3.12
+
+
+Abstract
+========
+
+This document describes the development and release schedule for
+Python 3.12.  The schedule primarily concerns itself with PEP-sized
+items.
+
+.. Small features may be added up to the first beta
+   release.  Bugs may be fixed until the final release,
+   which is planned for end of October 2023.
```

Comment:
It's planned for 2023-10-02, the _start_ of October 2023, but I think we can just say October 2023.

```suggestion
   which is planned for October 2023.
```